doc/ref_model/chapters/chapter03.md

```diff
@@ -180,9 +180,36 @@ The physical compute, storage and network resources serve as the foundation of t
 
+The following diagram represents a high level Network Model detailing, from the network perspective, major subcomponents of the Virtual Resources, Physical Resources, and Management Software components of the NFVI Model Overview in Figure 3-1.The details of this model are discussed in Section 4.3.
 
+<p align="center"><img src="../figures/RM-Chapter 3-Network-Model.png" alt="Network Model Overview" Title="Network Model Overview" width="65%"/></p>
+<p align="center"><b>Figure 3-5:</b> Network Model Overview</p>
+
+Principles that should be followed during the development and definition of the networking scope for the Reference Model, Reference Architectures, Reference Implementations and Reference Conformance test suites:
+
+• Abstraction: A standardized network abstraction layer between the Virtualisation Layers and the Network Physical Resources Layer that hides (or abstracts) the details of the Network Physical resources from the Virtualisation Layers.
+
+> **Note:**  In deployment phases this principle may be applied in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementations of pipeline stages and available platforms. The network abstraction layer supports, for example, physical resources with or without programmable hardware acceleration, or programmable network switches
+
+•Agnosticism: Define Network Fabric concepts and models that can carry any type of traffic in terms of:
+    Control, User and Management traffic types
+    Acceleration technologies supporting multiple types of infrastructure deployments and network function workloads
+    
+•Automation: Enable end-to-end automation, from Physical Fabric installation and provisioning to automation of workload onboarding.
+
+•Openness: All networking is based on open source or standardized APIs (North Bound Interfaces (NBI) and South Bound Interfaces (SBI)) and should enable integration of open source networking components (e.g. SDN controllers).
+
+•Programmability: Network model enables a programmable forwarding plane controlled from a separately deployed control plane.
+
+•Scalability: Network model enables scalability to handle all traffic traverse North-South and East-West enabling small up to large deployments.
+
+•Workload agnostic: Network model is capable of providing connectivity to any type of workloads, including VNF, CNF and BareMetal workloads.
+
+•Telco Grade: Network model is capable of supporting deployments of the telco grade workloads.
```

Comment:
Do we define what are telco grade workloads?
I think not only the workloads need to be telco grade, but the whole networking.